CMakeLists.txt

```diff
@@ -227,7 +232,14 @@ if(MI_BUILD_SHARED)
       COMMENT "Copy mimalloc-redirect.dll to output directory")
   endif()
 
-  install(TARGETS mimalloc EXPORT mimalloc DESTINATION ${mi_install_dir} LIBRARY)
+  if (MI_INSTALL_TOPLEVEL MATCHES "ON")
+    install(TARGETS mimalloc EXPORT mimalloc LIBRARY
+        RUNTIME DESTINATION bin
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
+  else()
+    install(TARGETS mimalloc EXPORT mimalloc DESTINATION ${mi_install_dir} LIBRARY)
+  endif()
```

Comment:
Is this logic necessary? What is installed in `bin` ?  If we can leave that in `lib` we can share the 
```
install(TARGETS mimalloc EXPORT mimalloc DESTINATION ${mi_install_dir} LIBRARY)
```
install always. If not, it is strange to even make `mi_install_dir` a parameter?

Reply:
`MI_INSTALL_TOPLEVEL=OFF`:
- `mi_install_dir = /usr/lib/mimalloc-1.6/`
- `mi_install_dir/mimalloc.so`
- `mi_install_dir/include/mimalloc.h`

`MI_INSTALL_TOPLEVEL=ON` with conda:
- `mi_install_dir = $CONDA_PREFIX`
- `mi_install_dir/lib/libmimalloc.so`
- `mi_install_dir/include/mimalloc.h`

We need this logic because on Conda `libmimalloc.so` is installed in `${mi_install_dir}/lib`, not `${mi_install_dir}`.

`RUNTIME DESTINATION bin` can be removed.

Reply:
Suggested by @alexreinking :

```
        RUNTIME DESTINATION bin
        LIBRARY DESTINATION lib
        ARCHIVE DESTINATION lib
```
to
```
        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}  // possibly lib64?
        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Reply:
Not `lib64`. You would set `CMAKE_INSTALL_LIBDIR=...lib64` when packaging a 64-bit (multiarch) build.